modules/apps/frontend-theme/frontend-theme-classic/src/WEB-INF/token-definition.json

```diff
@@ -0,0 +1,473 @@
+{
+	"tokenCategories": [
+		{
+			"label": "general",
+			"name": "general"
+		},
+		{
+			"label": "colors",
+			"name": "colors"
+		},
+		{
+			"label": "typography",
+			"name": "typography"
+		},
+		{
+			"label": "spaces",
+			"name": "spaces"
+		}
+	],
+	"tokenSets": [
+		{
+			"label": "general",
+			"name": "general"
+		},
+		{
+			"label": "utility",
+			"name": "utility"
+		},
+		{
+			"label": "layout",
+			"name": "layout"
+		},
+		{
+			"label": "buttons",
+			"name": "buttons"
+		},
+		{
+			"label": "alerts",
+			"name": "alerts"
+		},
+		{
+			"label": "cards",
+			"name": "cards"
+		},
+		{
+			"label": "forms",
+			"name": "forms"
+		},
+		{
+			"label": "nav",
+			"name": "nav"
+		},
+		{
+			"label": "portlet",
+			"name": "portlet"
+		},
+		{
+			"label": "tables",
+			"name": "tables"
+		},
+		{
+			"label": "lists",
+			"name": "lists"
+		}
+	],
+	"tokens": [
+		{
+			"ccp": "--white",
```

Comment:
Hey @izaera, just one question... I think we don't need the `--` here, right? CCP will all get `--` anyways and we're rendering the CSS Style, so I'd simply expect the CCP name and not the `--` prefix.

Is that correct?

Reply:
Yes, `ScopedCSSVariablesTopHeadDynamicInclude` [adds the `--`](https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-css/frontend-css-variables-web/src/main/java/com/liferay/frontend/css/variables/web/internal/servlet/taglib/ScopedCSSVariablesTopHeadDynamicInclude.java#L117) for the same reason.

Reply:
I'm going to remove it from the next commit! 😄